pkgs/development/python-modules/cmaes/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, hypothesis
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "cmaes";
+  version = "0.9.1";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "CyberAgentAILab";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-dd5vLT4Q0cI5ts0WgBpjPtOA81exGNjWSNHEiPggYbg=";
+  };
+
+  propagatedBuildInputs = [ numpy ];
+
+  nativeCheckInputs = [ pytestCheckHook hypothesis ];
+
+  pythonImportsCheck = [ "cmaes" ];
+
+  meta = with lib; {
+    description = "Python library for CMA evolution strategy";
+    homepage = "https://pypi.org/project/cmaes";
```

Comment:
```suggestion
    homepage = "https://github.com/CyberAgentAILab/cmaes";
```

Reply:
`format` added and changed the homepage.